internal/boxcli/run.go

```diff
@@ -41,6 +43,30 @@ func RunCmd() *cobra.Command {
 	return command
 }
 
+func InstallCmd() *cobra.Command {
+	flags := runCmdFlags{}
+	command := &cobra.Command{
+		Use:   "install",
+		Short: "Installs all packages mentioned in devbox.json",
+		Long: "Starts a new shell and installs all packages mentioned in devbox.json or a json file" +
```

Comment:
I don't think this is accurate:
> or a json file specified via --config

I believe we always look for a `devbox.json` file at the directory specified by `--config`. See these lines where we first get the "projectDir" and then append the hardcoded `configFilename` which is `devbox.json`:
https://github.com/jetpack-io/devbox/blob/bf52b9a6324f3d4fb65dc92ce1f67240f151cc9d/internal/impl/devbox.go#L88-L92